### PR TITLE
users can set a custom LCD server to not run into rate limiting issue…

### PR DESCRIPTION
**Pull request by mr-rooftop**
…s of lcd.terra.dev

### CODE REVIEW

File: bot/constants.py [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
-GOVERNANCE_PROPOSAL_ENDPOINT = 'http://localhost:8000/governance_proposals.json' if DEBUG else 'https://lcd.terra.dev/gov/proposals'
-NODE_INFO_ENDPOINT = 'http://localhost:8000/node_info.json' if DEBUG else 'https://lcd.terra.dev/node_info'
+GOVERNANCE_PROPOSAL_ENDPOINT = 'http://localhost:8000/governance_proposals.json' if DEBUG else f'https://{LCD_ENDPOINT}/gov/proposals'
+NODE_INFO_ENDPOINT = 'http://localhost:8000/node_info.json' if DEBUG else f'https://{LCD_ENDPOINT}/node_info'
 
 # Paths
 storage_path = os.sep.join([os.path.dirname(os.path.realpath(__file__)), os.path.pardir, 'storage'])
 session_data_path = os.sep.join([storage_path, 'session.data'])
 
-JOB_INTERVAL_IN_SECONDS = 30
```
Comment (mbartn) [inline bot/constants.py:38]: Is this change intentional?

**Review by mbartn — APPROVED** [review]

File: bot/constants.py (hunk 1 above)
Comment (mr-rooftop) [reply bot/constants.py:41]: yes
